highfidelity/hifi
# Show MyAvatar's head in the SpectatorCamera view!

Woohoo!!!

**Test Plan:**
1. Enter HMD mode
2. Enable the Spectator Camera using the Tablet App
3. Point the camera at your face. Verify that your face shows up in the camera's viewfinder preview.
4. Enter Desktop mode
5. Go into third person mode. Verify that your head still renders.
6. Go into first person mode. Verify that your head doesn't render (i.e. you can't see the inside of your head in first person mode)
